design/one-pager-scope-workload-interaction-mechanism.md

```diff
@@ -0,0 +1,141 @@
+# Scopes and workloads interaction mechanism in OAM
```

Comment:
This is very similar to the trait-workload interaction, how about just modify that proposal instead?

Reply:
maybe I can follow Ryan's idea 
- just modify the existing proposal instead of a new one, 

and follow @wonderflow advice to 
- write a separate section from Trait's in the proposal.
- narrow the problem statement to be `current implementation is hard code` issue

Reply:
I have merged this proposal into existing one. Once you check it as okay, I will remove the separate one and rename the existing one, e.g. "one-pager-trait-scope-workload-interaction-mechanism.md"